thatkitebot/cogs/electronics.py

```diff
@@ -48,6 +48,40 @@ def slash_preprocessor(a: str):
     """
     return a.replace("v", "").replace("V", "").replace("u", "µ").replace("F", "").strip() if a else None
 
+def FAQ():
+    embed = discord.Embed(title = "How can I start learning electronics?", color = discord.Color.gold())
+    embed.add_field(name="Youtube", 
+        value="""You should start here:
+                [RSD Academy](https://www.youtube.com/c/RSDAcademy)
+                [Leo's Bag of Tricks](https://www.youtube.com/channel/UCe1bjEcBichpiAMhExh0NiQ)
+                DIY projects with good explanations:
+                [GreatScott!](https://www.youtube.com/c/greatscottlab)
+                [Electronoobs](https://www.youtube.com/c/ELECTRONOOBS)
+                Good source of entertainment and education:
+                [Electroboom](https://www.youtube.com/c/Electroboom/)
+                [DiodeGoneWild](https://www.youtube.com/c/DiodeGoneWild/)
+                 """)
+    embed.add_field(name="Youtube",
```

Comment:
why have an extra youtube field if the first field is also just youtube links

Reply:
embed value can only have 1024 characters

Reply:
:raised_eyebrow:

Reply:
`discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body In embeds.0.fields.0.value: Must be 1024 or fewer in length.`